### ISSUE TITLE
Use of a vulnerable dependency

### ISSUE THREAD

**Issue by Berkmann18** [cheeriojs/cheerio#1270]
Snyk recently pointed out in some of my projects that some dependencies were vulnerable to an issue that goes back to `extend` and `sshpk` but those were apparently fixed in the `request@2.88.0` which has been updated on `jsdom` (currently at v13.1.0).
This package, however, doesn't use an up-to-date and vulnerability free version of `jsdom` and PRs like #1230 are still hanging.
Would it be possible to get this sorted (I was planning on opening a PR myself but @greenkeeper already did)?

**Comment by robogeek** [comment]
I'm curious if the Cheerio project is still alive.  The last commit to the README claims that ongoing development is happening in the main branch, and gives a link for the branch for the last release -- however that commit was a few months ago and of course there's been no activity since.

I find this worrying since I've built a fairly large application (https://akashacms.com) around Cheerio.

**Comment by Berkmann18** [comment]
@robogeek Yeah, the last commit was 4 months ago but maybe the gitter chat might be more active.

**Comment by Berkmann18** [comment]
> What I'm concerned about is bug fixing activity, not discussions

So am I.

>  I did find the Gitter chat and while there is discussion I don't know if I'd call it "active".

Hum, interesting :s.
